topics/statistics/tutorials/RNN/workflows/Intro_To_RNN_v1.0.10.0.ga

```diff
@@ -0,0 +1,368 @@
+{
+    "a_galaxy_workflow": "true",
```

Comment:
fyi there are more lintung errors which you'll find in the Files tab of GitHub. Thanks for fixing them!